### PR TITLE
[docs] Update docs for topic stats in admin apis.

### PR DESCRIPTION
**Pull request by Jason918**
### Motivation

There are a lot new variables which are not explained in docs when we get topic stats data using the cmd `bin/pulsar-admin topics stats <topic>`.
This PR is trying to sync the docs with the source code.

### Modifications

The new docs mostly come from comment in source code.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

Only doc is updated.

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API: ( no)
  - The schema: ( no )
  - The default values of configurations: (no)
  - The wire protocol: ( no)
  - The rest endpoints: ( no)
  - The admin cli options: (no)
  - Anything that affects deployment: (no )

### Documentation

  - Does this pull request introduce a new feature? ( no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by Jason918** [comment]
@Anonymitaet Hi, can you please take a look?

File: site2/docs/admin-api-topics.md [hunk 1]
```diff
@@ -187,14 +187,42 @@ You can check the following statistics of a given non-partitioned topic.
 
   -   **storageSize**: The sum of the ledgers' storage size for this topic. The space used to store the messages for the topic.
 
+  -   **bytesInCounter**: Total bytes published to the topic (bytes).
```
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:190]:
```suggestion
  -   **bytesInCounter**: Total bytes published to the topic.
```

File: site2/docs/admin-api-topics.md [hunk 2]
```diff
@@ -187,14 +187,42 @@ You can check the following statistics of a given non-partitioned topic.
...
+
+  -   **msgInCounter**: Total messages published to the topic (msg).
```
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:192]:
```suggestion
  -   **msgInCounter**: Total messages published to the topic.
```

File: site2/docs/admin-api-topics.md [hunk 3]
```diff
@@ -187,14 +187,42 @@ You can check the following statistics of a given non-partitioned topic.
...
+
+  -   **bytesOutCounter**: Total bytes delivered to consumer (bytes).
+
+  -   **msgOutCounter**: Total messages delivered to consumer (msg).
```
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:196]:
```suggestion
  -   **msgOutCounter**: Total messages delivered to consumer.
```

File: site2/docs/admin-api-topics.md [hunk 4]
```diff
@@ -187,14 +187,42 @@ You can check the following statistics of a given non-partitioned topic.
...
+
+  -   **msgChunkPublished**: Topic has chunked message published on it.
+
+  -   **backlogSize**: Estimated total unconsumed or backlog size in bytes.
```
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:200]:
```suggestion
  -   **backlogSize**: Estimated total unconsumed or backlog size (in bytes).
```

File: site2/docs/admin-api-topics.md [hunk 5]
```diff
@@ -187,14 +187,42 @@ You can check the following statistics of a given non-partitioned topic.
...
+
+  -   **offloadedStorageSize**: Space used to store the offloaded messages for the topic.
+  
+  -   **waitingPublishers**: The number of publishers waiting in queue in exclusive access mode.
```
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:204]:
```suggestion
  -   **waitingPublishers**: Number of publishers waiting in a queue in exclusive access mode.
```

File: site2/docs/admin-api-topics.md [hunk 6]
```diff
@@ -219,12 +251,46 @@ You can check the following statistics of a given non-partitioned topic.
...
           
-          -   **lastConsumedFlowTimestamp**: The timestamp of the last flow command received. 
-          
+          -   **lastConsumedFlowTimestamp**: The timestamp of the last flow command received.
+
           -   **lastConsumedTimestamp**: The latest timestamp of all the consumed timestamp of the consumers.
           
           -   **lastAckedTimestamp**: The latest timestamp of all the acked timestamp of the consumers.
 
+          -   **bytesOutCounter**: Total bytes delivered to consumer (bytes).
```
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:260]:
```suggestion
          -   **bytesOutCounter**: Total bytes delivered to consumer.
```

File: site2/docs/admin-api-topics.md [hunk 7]
```diff
@@ -219,12 +251,46 @@ You can check the following statistics of a given non-partitioned topic.
...
+
+          -   **msgOutCounter**: Total messages delivered to consumer (msg).
```
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:262]:
```suggestion
          -   **msgOutCounter**: Total messages delivered to consumer.
```

File: site2/docs/admin-api-topics.md [hunk 8]
```diff
@@ -219,12 +251,46 @@ You can check the following statistics of a given non-partitioned topic.
...
+
+          -   **msgRateRedeliver**: Total rate of messages redelivered on this subscription (msg/s).
+
+          -   **chunkedMessageRate**: Chunked message dispatch rate.
+
+          -   **backlogSize**: Size of backlog in byte.
```
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:268]:
```suggestion
          -   **backlogSize**: Size of backlog (in byte).
```

File: site2/docs/admin-api-topics.md [hunk 9]
```diff
@@ -219,12 +251,46 @@ You can check the following statistics of a given non-partitioned topic.
...
+
+          -   **msgBacklogNoDelayed**: Number of messages in the subscription backlog that do not contain the delay messages.
+
+          -   **blockedSubscriptionOnUnackedMsgs**: Flag to verify if subscription is blocked due to reaching threshold of unacked messages.
```
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:272]:
```suggestion
          -   **blockedSubscriptionOnUnackedMsgs**: Flag to verify if a subscription is blocked due to reaching threshold of unacked messages.
```

File: site2/docs/admin-api-topics.md [hunk 10]
```diff
@@ -219,12 +251,46 @@ You can check the following statistics of a given non-partitioned topic.
...
+
+          -   **msgDelayed**: Number of delayed messages currently being tracked.
+
+          -   **unackedMessages**: Number of unacknowledged messages for the subscription.
+
+          -   **activeConsumerName**: The name of the consumer that is active for single active consumer subscriptions i.e. failover or exclusive.
```
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:278]:
```suggestion
          -   **activeConsumerName**: The name of the consumer that is active for single active consumer subscriptions. For example, failover or exclusive. 
```

File: site2/docs/admin-api-topics.md [hunk 11]
```diff
@@ -219,12 +251,46 @@ You can check the following statistics of a given non-partitioned topic.
...
+
+          -   **unackedMessages**: Number of unacknowledged messages for the subscription.
+
+          -   **activeConsumerName**: The name of the consumer that is active for single active consumer subscriptions i.e. failover or exclusive. 
+
+          -   **totalMsgExpired**: Total messages expired on this subscription. 
+
+          -   **lastMarkDeleteAdvancedTimestamp**: Last MarkDelete position advanced timestamp.
+
+          -   **durable**: Tells whether this subscription is durable or ephemeral (eg.: from a reader).
```
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:284]:
```suggestion
          -   **durable**: Whether the subscription is durable or ephemeral (for example, from a reader).
```

File: site2/docs/admin-api-topics.md [hunk 12]
```diff
@@ -239,10 +305,32 @@ You can check the following statistics of a given non-partitioned topic.
...
+
                 -   **lastAckedTimestamp**: The timestamp when the consumer acknowledges a message the last time.
 
+                -   **address**: The IP address and source port for the connection of this consumer.
+
+                -   **connectedSince**: The timestamp when this consumer is created or reconnected last time.
+
+                -   **clientVersion**: The client library version of this consumer.
+
+                -   **bytesOutCounter**: Total bytes delivered to consumer (bytes).
```
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:318]:
```suggestion
                -   **bytesOutCounter**: Total bytes delivered to consumer.
```

File: site2/docs/admin-api-topics.md [hunk 13]
```diff
@@ -239,10 +305,32 @@ You can check the following statistics of a given non-partitioned topic.
...
+
+                -   **msgOutCounter**: Total messages delivered to consumer (msg).
```
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:320]:
```suggestion
                -   **msgOutCounter**: Total messages delivered to consumer.
```

**Review by Anonymitaet — COMMENTED** [review]
Thanks for your contribution. I’ve left some comments, PTAL.

**Comment by Jason918** [comment]
> Thanks for your contribution. I’ve left some comments, PTAL.

@Anonymitaet  Thx, all the comments are resolved, PTAL.

**Review by Anonymitaet — APPROVED** [review]

**Review by michaeljmarshall — COMMENTED** [review]
Left some minor comments. This is a great addition.

File: site2/docs/admin-api-topics.md [hunk 14]
```diff
@@ -187,14 +187,42 @@ You can check the following statistics of a given non-partitioned topic.
...
+
+  -   **bytesOutCounter**: Total bytes delivered to consumer (bytes).
+
+  -   **msgOutCounter**: Total messages delivered to consumer.
+
+  -   **msgChunkPublished**: Topic has chunked message published on it.
+
+  -   **backlogSize**: Estimated total unconsumed or backlog size (in bytes).
+
+  -   **offloadedStorageSize**: Space used to store the offloaded messages for the topic.
```
[the remark this answers is not in the document]
Comment (Jason918) [reply site2/docs/admin-api-topics.md]: Add **bytes** in the latest commit.

**Comment by Jason918** [comment]
@michaeljmarshall Thank you for your comments. All the comments are resolved, PTAL.

**Comment by Jason918** [comment]
@Anonymitaet Hi, Anyone we should invite to review to close this PR?

**Comment by Anonymitaet** [comment]
@Jason918 I think now it's fine, we can merge this PR after it passes the tests.

**Comment by Jason918** [comment]
> @Jason918 I think now it's fine, we can merge this PR after it passes the tests.

@Anonymitaet  Great, thanks a lot~

**Pull request merged**